Images.yaml

```diff
@@ -0,0 +1,56 @@
+type: object
+required:
+  - id
+  - file_name
+  - license
+  - agcontext_id
+  - width
+  - height
+  - resolution
+
+properties:
+
+  id:
+    type: number
+    description: |-
+      A number identifying the image.
+
+  file_name:
+    type: string
+    description: |-
+      A string identifying the filename of the image.
+      This string does not include the path of the image.
```

Comment:
I assume it can include the path of the image relative to the image base?

Or could it be a URL in the general case?